examples/street-view-panorama-map/README.md

```diff
@@ -0,0 +1,34 @@
+# `useStreetViewPanorama` Hook Setup Example
+
+This is an example setup of **Street View Panorama** to show the usage of the **useStreetViewPanorama hook** without passing a div element with the Google Maps React Hooks library.
```

Comment:
I think most of the readers won't know what "without passing a div element" means in this case.
```suggestion
This is an example setup of **Street View Panorama** to show the usage of the **useStreetViewPanorama hook** to enable Street View in an existing map with the Google Maps React Hooks library.
```

Reply:
Yes sounds good 👍